src/components/BookList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Book from './Book'
+import CreateNewBook from './CreateNewBook ';
+
+const BookList = () => (
+  <div className="container max-width-md">
+    <Book title="Hoom Hoom" author="Blah 2"></Book>
+    <Book title="Zing Zing" author="Blah 2"></Book>
+    <Book title="Bazingazing" author="Blah 2"></Book>
+    <CreateNewBook></CreateNewBook>
```

Comment:
**Changes Required**
Good effort implementing the requested changes by the previous Code Reviewer. However;
- [ ] The call to the CreateNewBook component should be self closing. For instance, `<CreateNewBook />`. This is one of React's good practice. Let us please try and change to this.

Reply:
Hey @oxenprogrammer can you have at my submission.